server/chart_config/climate.json

```diff
@@ -10,5 +10,19 @@
       "scale": false
     },
     "isChoropleth": true
+  },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-Daily_max_temperature_rcp85_diff_to_2006",
+    "title": "Daily max temperature change (from year 2006)",
+    "description": "Daily max temperature change from year 2006 based on RCP 8.5",
+    "statsVars": [
+      "DifferenceRelativeToBaseDate2006_Daily_Max_Temperature_RCP85"
+    ],
+    "unit": "K",
```

Comment:
this usually needs additions to units.json (which was manually curated :-/), but looks like we get lucky here :)

Reply:
Got. So only default symbol is used, we don't need to add it to units.json?

Reply:
Yea, if we want to keep it as "K"

Maybe we might want to do that as a follow up anyway, since K isn't a commonly used, and it could be confused for "thousands" the way it's displayed now. wdyt? maybe we should spell it out

Reply:
Changed to "celsius" and use "°c" as symbol